packages/MarkdownEditor-Tests.package/MarkdownHeadingsTest.class/instance/testMatchHeadingLevels1To6WithText.st

```diff
@@ -0,0 +1,6 @@
+tests - matching
+testMatchHeadingLevels1To6WithText
+	
+	#('# MiniMetro' '## the fastest game' '### on the planet!' '#### cool play style' 
```

Comment:
Oi, I think this will lead to another polarized discussion so here goes nothing:
I find the idea of MiniMetro-flavored test strings funny, but it adds nothing to the understanding of the tests. Instead, they just make the tests wordy.
I would strive for either

- self-explaining test strings like https://github.com/hpi-swa-teaching/MarkdownEditor/blob/3448307b24dbe24d713d603e2018d25a12dcf931/packages/MarkdownEditor-Tests.package/MarkdownBlockParserTest.class/instance/testBlockStructureParagraphWithHeading.st#L6
- or boring, but short test strings like
  ```smalltalk
  {'# H1'. '##  H2'. ... }
  ```

Reply:
```smalltalk
isParsedStructureOf: 'Paragraph which is interrupted by', Character cr, '# Train arriving'
```
😋

Reply:
I don't think we need to be that serious in the tests. Having a bit of fun with them is nice. Thus I like the Minimetro tests.